### ISSUE TITLE
XSS

### ISSUE THREAD

**Issue by Sparox** [zHaytam/SmartBreadcrumbs#50]
Hello,

I've noticed that string passed to the constructor `MvcBreadcrumNode` don't prevent xss attacks. 
(Maybe because they're not escaped before rendering)

My workaround is to encapsulate my string with HttpUtility.HtmlEncode(string) but it's a tedious process.
Is there a workaround built-in your library?

Thank you very much

The escaped version: 
![image](https://user-images.githubusercontent.com/6002838/71994002-4bd48e00-3238-11ea-95c3-b5ad9cab89bc.png)

Without workaround:
![image](https://user-images.githubusercontent.com/6002838/71994462-11b7bc00-3239-11ea-9ced-2f3b39e6556b.png)
![image](https://user-images.githubusercontent.com/6002838/71994524-2e53f400-3239-11ea-98a5-f79a72cb5571.png)

**Comment by zHaytam** [comment]
Isn't that the tile of the page?

**Comment by zHaytam** [comment]
No string is escaped in the library since I thought everything would be manually set by the developer, can you give me your scenario please?

**Comment by Sparox** [comment]
Sure, my application (web application with netcore) permits to create objects such as application, equipment or whatever.
The user can enter a name, a description and few informations for each objects.
(The creation has this breadcrumb for example: Home > Applications >  New Application)

Then I have a listing of objects with this breadcrumb:
Home > Applications 

Finally, the user can view the object created (basically a simple CRUD) with this expected breadcrumb: Home > Applications > "Name of application".

An application's name can have special characteres such as ',",<, >, etc. 
All labels and fields in aspnet prevent xss attacks (all displayed strings are escaped automagically, as explained in this doc https://docs.microsoft.com/en-us/aspnet/core/security/cross-site-scripting?view=aspnetcore-3.1#html-encoding-using-razor) and if I remove the Breadcrumb of my view, the script injected isn't executed.

**To summarize, I want to be able to display strings from end users in the breadcrumb without any risk of injection of any sort.**

**Comment by zHaytam** [comment]
Now I understand. I will fix this issue asap!

**Comment by zHaytam** [comment]
Hi, this is now fixed in 3.1, download the new version for NuGet!
